RecoVertex/BeamSpotProducer/plugins/OfflineToTransientBeamSpotESProducer.cc

```diff
@@ -0,0 +1,37 @@
+#include "RecoVertex/BeamSpotProducer/plugins/OfflineToTransientBeamSpotESProducer.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Utilities/interface/do_nothing_deleter.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+using namespace edm;
+
+OfflineToTransientBeamSpotESProducer::OfflineToTransientBeamSpotESProducer(const edm::ParameterSet& p) {
+  auto cc = setWhatProduced(this);
+
+  bsOfflineToken_ = cc.consumesFrom<BeamSpotObjects, BeamSpotObjectsRcd>();
+}
+
+OfflineToTransientBeamSpotESProducer::~OfflineToTransientBeamSpotESProducer() {
+  //delete theOfflineBS_;
+}
+void OfflineToTransientBeamSpotESProducer::fillDescription(edm::ConfigurationDescriptions& desc) {
```

Comment:
This should be 
```suggestion
void OfflineToTransientBeamSpotESProducer::fillDescriptions(edm::ConfigurationDescriptions& desc) {
```
to have any effect.